### ISSUE TITLE
Dead Ahead is broken.

### ISSUE THREAD

**Issue by Mackaral132** [VenomiXeD/DeadAheadIssues#3]
**Describe the bug**
Dead Ahead is either broken or my pc is crap. So anyways music isnt playing. When I join it plays but the game seems to have trouble switching between soundtracks. Also secondary guns are super beefy. By that I mean two shells from a secondary gun from a orion destroyer ship sank a BB (Battleship) The BB wasn't already super damaged because our torpedoes where afk. Also when ships sink, they roll around in the water like crazy and start flinging people in all directions. Also in Dreadnought the orion torpedoes launchers only fire one torpedo instead of 4. Also I can see the usernames of the other team sometimes and its kinda unfair to them as I know their whereabouts part of the time. It glitches.
![e](https://user-images.githubusercontent.com/96366094/146665704-7be95d04-fdb9-41dd-b6ae-33fcc8eafdf1.png)
![2021-12-18 (2)](https://user-images.githubusercontent.com/96366094/146665710-1bfd34a4-89ea-4de3-985b-c22c01fef5da.png)
![2021-12-18 (1)](https://user-images.githubusercontent.com/96366094/146665714-4e445327-24ab-47a4-b185-2cdea41a8126.png)

**To Reproduce**
Happens randomly

**Expected behavior**
Roblox crashes almost every time this happens.

**Screenshots**

**Additional context**
My pc may be crap but I asked other players and they said they saw it too. Also I took the screenshot too late of the people getting flinged but you can still see one person in the sky. And no that is not a plane.

**Comment by Mackaral132** [comment]
The first image is the username glitch, the second image is the flinging glitch and the third image is the torpedo glitch.

**Comment by Mackaral132** [comment]
Oh and here is the video of no music playing.

https://user-images.githubusercontent.com/96366094/146665825-2755b02f-6220-49f4-8dee-f32f46c99503.mp4

**Comment by VenomiXeD** [comment]
Yes hello, I am not sure what is causing this issue, we'll look over it.

**Comment by YourLocalDave** [comment]
I wonder how this guy had so many issues; he must be either some really unlucky guy or a glitch hunter.

**Comment by Mackaral132** [comment]
I am just a really unlucky guy.
